Proposal-Process.md

```diff
@@ -0,0 +1,14 @@
+For a proposal to create or change a process foundation-wide, a PR should be
```

Comment:
Minor nit: Can you add a header to this document? `## Policy Change Proposal Process` or something along those lines